webpack/scenes/RedHatRepositories/components/SearchBar.js

```diff
@@ -97,6 +97,7 @@ class SearchBar extends Component {
         <MultiSelect
           value={this.state.filters}
           options={filterOptions}
+          noneSelectedText="Filter by repo type"
```

Comment:
This should be translated with `__()` here.

Reply:
Would it it be too long to use 'repository' versus 'repo'?  It may be better for consistency and translations.